padle/padle.py

```diff
@@ -252,20 +253,18 @@ async def stats(self, ctx):
                 correct_id = await self.config.padle_today()
             else:
                 correct_id = save_daily[day - 1][0]
-            if str(day) in all_guesses and len(all_guesses[str(day)]) != 0 and all_guesses[str(day)][-1] == correct_id:
+            if str(day) in all_guesses and correct_id in all_guesses[str(day)]:
```

Comment:
`if correct_id in all_guesses.get(str(day), ()):`, though that would create a tuple each time.

Does this load the user's entire guess history and reprocess it? Can a user store more than 6 guesses per day? Can someone spam guesses one day and cause Tsubotki to do a huge load whenever `^stats` is used? (Probably not a real concern.)